pymatgen/analysis/diffraction/tem.py

```diff
@@ -107,7 +107,7 @@ def generate_points(coord_left: int = -10, coord_right: int = 10) -> np.ndarray:
     def zone_axis_filter(
         self, points: list[tuple[int, int, int]] | np.ndarray, laue_zone: int = 0
     ) -> list[tuple[int, int, int]]:
-        """Filters out all points that exist within the specified Laue zone according to the zone axis rule.
+        """Filter out all points that exist within the specified Laue zone according to the zone axis rule.
```

Comment:
Thanks for reinforcing this, I fixed some similar issues (and thought `pymatgen` has a different convention than others). PEP-8 also seems to support this style.